sdk/go/databricks/clusterPolicy.go

```diff
@@ -51,25 +51,26 @@ import (
 //
 //	func main() {
 //		pulumi.Run(func(ctx *pulumi.Context) error {
-//			_ := map[string]interface{}{
+//			personalVmOverride := map[string]interface{}{
 //				"autotermination_minutes": map[string]interface{}{
 //					"type":   "fixed",
 //					"value":  220,
 //					"hidden": true,
 //				},
 //				"custom_tags.Team": map[string]interface{}{
 //					"type":  "fixed",
-//					"value": _var.Team,
+//					"value": team,
 //				},
 //			}
-//			tmpJSON0, err := json.Marshal(personal_vm_override)
```

Comment:
Nice fix. Team is still missing, but this is much better.

Reply:
I have a mechanism we could try https://github.com/pulumi/pulumi-terraform-bridge/pull/1844 for filling out missing variables of known type; may or may not apply here.